stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/processor/local/LocalGovActionStateProcessor.java

```diff
@@ -54,13 +55,19 @@ public void handleEpochChangeEvent(EpochChangeEvent epochChangeEvent) {
         if (!syncMode)
             return;
 
+        localGovStateService.setEra(Era.valueOf(epochChangeEvent.getEra().name()));
```

Comment:
Can we move both blockEvent and epochEvent from this class to `LocalDRepDistrService` class to align with LocalEpochParam? Then we don't need to set era from outside. The processor can only have `rollback` method.

Reply:
Also, let's try to avoid  a separate field for `syncMode` as this is already available in event metadata.

Reply:
> Also, let's try to avoid a separate field for `syncMode` as this is already available in event metadata.
Yes, but in processor class, I think we still need 'syncMode' field since rollbackEvent don't have event metadata

Reply:
Got it. Thanks.